site/src/pages/docs/advanced-custom-attributes.mdx

```diff
@@ -36,3 +36,35 @@ ctx.attrs().forEachRemaining(e -> {
     System.err.println(e.getKey() + ": " + e.getValue());
 });
 ```
+
+## Setting attributes using <type://RequestOptions>
```

Comment:
How about splitting this into two sections, one for using preparation and the other for using options?